Example/Pods/Local Podspecs/TABTestKit.podspec.json

```diff
@@ -8,9 +8,10 @@
     "file": "LICENSE"
   },
   "authors": {
-    "zacoid55": "zac@theappbusiness.com",
-    "KaneCheshire": "kane.cheshire@theappbusiness.com",
-    "theblixguy": "suyash.srijan@theappbuisness.com"
+    "zacoid55": "zac.borrelli@kinandcarta.com",
+    "KaneCheshire": "kane.cheshire@kinandcarta.com",
+    "theblixguy": "suyash.srijan@theappbusiness.com",
```

Comment:
Suyash is currently on furlough but do we want to update his domain address also?

Reply:
Was having trouble finding it, but I've fixed now